docs/src/components/home/ProductsSwitcher.tsx

```diff
@@ -10,6 +12,8 @@ import IconImage from 'docs/src/components/icon/IconImage';
 import KeyboardArrowRightRounded from '@material-ui/icons/KeyboardArrowRightRounded';
 import ROUTES from 'docs/src/route';
 
+const SwipeableViews = dynamic(() => import('react-swipeable-views'), { ssr: false });
```

Comment:
Oh, we use my component 😅, pretty crazy that I abandoned it 3 years ago and that the community continue adopting it.

Reply:
I use it because I don't want to introduce new library and what we are using for rebranding is common, so I think we can live with it?.

Reply:
Ok, I'm resolving